package.json

```diff
@@ -12,24 +12,26 @@
     "async": "^1.5.0",
     "compression": "^1.x",
     "cors": "^2.x",
-    "errorhandler": "^1.x",
-    "loopback": "^2.14.0",
+    "loopback": "^3.x",
     "loopback-boot": "^2.6.5",
-    "loopback-datasource-juggler": "^2.19.0",
-    "serve-favicon": "^2.0.1"
+    "serve-favicon": "^2.0.1",
+    "strong-error-handler": "^1.x"
   },
   "devDependencies": {
-    "eslint": "^1.x",
+    "eslint": "^5.16.0",
```

Comment:
Is it necessary to update `eslint`?

Reply:
For posterity, as discussed, this is because of security vulnerabilities reported by `npm audit`.